dev-packages/browser-integration-tests/suites/integrations/featureFlags/openfeature/basic/test.ts

```diff
@@ -4,7 +4,7 @@ import { sentryTest } from '../../../../../utils/fixtures';
 
 import { envelopeRequestParser, shouldSkipFeatureFlagsTest, waitForErrorRequest } from '../../../../../utils/helpers';
 
-const FLAG_BUFFER_SIZE = 100; // Corresponds to constant in featureFlags.ts, in browser utils.
+import { FLAG_BUFFER_SIZE } from '../../constants';
```

Comment:
This is fine for this PR, but in the future it'd be better practice to introduce the constant in this PR and do the refactor for the other integrations in a separate PR.